.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,41 @@
+name: Pull request
+
+on:
+    push:
+        branches:
+            - master
```

Comment:
yes, this makes sense. `master` is write protected anyway so anytime master is pushed to, it came from a pull request.

Reply:
Thank you, Zack!